charts/prometheus-thanos/values.yaml

```diff
@@ -67,6 +67,24 @@ querier:
     successThreshold: 1
     timeoutSeconds: 30
 
+  autoscaling:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 10
+    metrics:
```

Comment:
Could you please add a reference to the releated k8s documentation about HPA?

Reply:
Added, thanks.